src/httpclient.cc

```diff
@@ -1,6 +1,8 @@
 #include "httpclient.h"
 
 #include <assert.h>
+#include <libp11.h>
```

Comment:
IWe need to put this behind the #ifdef BUILD_P11

I think that we will also need to make one of the test builds run without libp11 installed, to check that the build without libp11 works

Reply:
How about running test.sh without BUILD_P11 then?

Reply:
My guess is that libp11.h gets installed in a location on the default include search path, so the build will succeed on any machine with the libp11 debian package installed.

Reply:
So do we need a separate Dockerfile to test that?